### PR TITLE
Gracefully handle unexpected task output

### PR DESCRIPTION
**Pull request by mhuth**
Before patch to ExpandedIssues:
Screenshot from 2023-11-22 14-03-50

After patch to ExpandedIssues:
Screenshot from 2023-11-22 14-04-33

### CODE REVIEW

**Comment by mhuth** [comment]
Found another problems.  Before patch to JobResultsDetails:
Screenshot from 2023-11-22 18-46-29

After patch to JobResultsDetails:
Screenshot from 2023-11-22 18-53-14

**Comment by mhuth** [comment]
To access the reproducers for these fixes, login as Insights QA user and try to view these completed tasks.  They will fail with the errors shown in the screenshots earlier in the PR.

Screenshot from 2023-11-23 12-15-59

File: src/SmartComponents/CompletedTaskDetails/JobResultsDetails/ExpandedIssues.js [hunk 1]
```diff
@@ -73,7 +73,11 @@ const ExpandedIssues = ({ rowPairs, isReportJson }) => {
             backgroundColor: '#ffffff',
           }}
         >
-          <CodeBlockCode>{rowPairs}</CodeBlockCode>
+          <CodeBlockCode>
+            {rowPairs instanceof Object
```
Comment (mkholjuraev) [inline src/SmartComponents/CompletedTaskDetails/JobResultsDetails/ExpandedIssues.js:77]: instanceof Object is not recommended by the JS community as it fails to properly check the type. Reference: https://bobbyhadz.com/blog/javascript-check-if-value-is-object#dont-use-the-instanceof-operator-to-check-if-a-value-is-an-object. Can we use another assertion?

**Review by mkholjuraev — COMMENTED** [review]
LGTM! I have verified that those tasks are now fixed. One possible improvement though.

File: src/SmartComponents/CompletedTaskDetails/JobResultsDetails/ExpandedIssues.js (hunk 1 above)
Comment (mhuth) [reply src/SmartComponents/CompletedTaskDetails/JobResultsDetails/ExpandedIssues.js]: Thanks for the feedback @mkholjuraev.  How about the typeof operator?

File: src/SmartComponents/CompletedTaskDetails/JobResultsDetails/ExpandedIssues.js [hunk 2]
```diff
@@ -73,7 +73,11 @@ const ExpandedIssues = ({ rowPairs, isReportJson }) => {
             backgroundColor: '#ffffff',
           }}
         >
-          <CodeBlockCode>{rowPairs}</CodeBlockCode>
+          <CodeBlockCode>
+            {typeof rowPairs === 'object'
```
Comment (mkholjuraev) [inline src/SmartComponents/CompletedTaskDetails/JobResultsDetails/ExpandedIssues.js:77]: We need to add 2 more conditions if we use `typeof`, [reference](https://bobbyhadz.com/blog/javascript-check-if-value-is-object#check-if-a-value-is-an-object-in-javascript). This one of the works confusing aspects of JS :)
```suggestion
            {typeof rowPairs === 'object' &&  rowPairs !== null &&
  !Array.isArray(rowPairs)
```

Comment (mhuth) [reply src/SmartComponents/CompletedTaskDetails/JobResultsDetails/ExpandedIssues.js]: Thanks @mkholjuraev.  If a tasks returns a null or an array then I think we'll get a similar Sentry error, but I guess we can cross that bridge when we come to it.

**Review by mkholjuraev — APPROVED** [review]
LGTM! thank you @mhuth

**Pull request merged**